packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -188,19 +193,27 @@ const AccountSignup = ({
                                     </div>
                                 </div>
                             ) : (
-                                <PasswordSelectionModal
-                                    api_error={api_error}
-                                    errors={errors}
-                                    handleBlur={handleBlur}
-                                    handleChange={handleChange}
-                                    isModalVisible={isModalVisible}
-                                    isSubmitting={isSubmitting}
-                                    touched={touched}
-                                    pw_input={pw_input}
-                                    setFieldTouched={setFieldTouched}
-                                    updatePassword={updatePassword}
-                                    values={values}
-                                />
+                                <React.Fragment>
+                                    <PasswordSelectionModal
+                                        api_error={api_error}
+                                        errors={errors}
+                                        handleBlur={handleBlur}
+                                        handleChange={handleChange}
+                                        isModalVisible={isModalVisible}
+                                        isSubmitting={isSubmitting}
+                                        touched={touched}
+                                        pw_input={pw_input}
+                                        setFieldTouched={setFieldTouched}
+                                        updatePassword={updatePassword}
+                                        values={values}
+                                    />
+                                    {is_questionnarie && (
+                                        <QuestionnarieModal
+                                            ab_questionnarie={ab_questionnarie}
```

Comment:
```suggestion
                                            ab_questionnaire={ab_questionnarie}
```
please fix this typo in every such word in this PR